pep-9999.rst

```diff
@@ -0,0 +1,89 @@
+PEP: 9999
+Title: Recording provenance of installed packages
+Author: Fridolin Pokorny <fridolin.pokorny at datadoghq.com>, Trishank Karthik Kuppusamy <trishank.kuppusamy at datadoghq.com>, Donald Stufft <donald.stufft at datadoghq.com>, Ofek Lev <ofek.lev at ofek.lev@datadoghq.com datadoghq.com>
+Sponsor: Paul Moore <p.f.moore at gmail.com>
+PEP-Delegate:
+Discussions-To: TODO
+Status: Draft
+Type: Process
+Content-Type: text/x-rst
+Created: 30-Jan-2002
+Post-History:
+
+
+Abstract
+========
+
+This PEP describes a way to record provenance of Python packages installed.
+This record is created by an installer and is available to users in a form of a
+JSON file ``direct_url.json`` in ``.dist-info`` directory. This PEP extends
+`PEP-610 <https://peps.python.org/pep-0610/>`_.
+
+
+Motivation
```

Comment:
One important thing from our discussion: what does this proposal give you that installation reports already don't? One may argue that installation reports are too verbose and too specific to the OS/Python versions. However, I think the more significant philosophical difference is that you want these reports/provenance to be produced by default, since most end-users are not going to go through the trouble to explicitly produce them unless compelled. However, even if these reports are produced by default, the problem remains that old applications/containers are going to be missing these reports.